hnn_core/cell_response.py

```diff
@@ -393,3 +394,58 @@ def write(self, fname):
                         self._spike_times[trial_idx][spike_idx],
                         int(self._spike_gids[trial_idx][spike_idx]),
                         self._spike_types[trial_idx][spike_idx]))
+
+
+def read_spikes(fname, gid_ranges=None):
+    """Read spiking activity from a collection of spike trial files.
+
+    Parameters
+    ----------
+    fname : str
+        Wildcard expression (e.g., '<pathname>/spk_*.txt') of the
+        path to the spike file(s).
+    gid_ranges : dict of lists or range objects | None
+        Dictionary with keys 'evprox1', 'evdist1' etc.
+        containing the range of Cell or input IDs of different
+        cell or input types. If None, each spike file must contain
+        a 3rd column for spike type.
+
+    Returns
+    ----------
+    cell_response : CellResponse
+        An instance of the CellResponse object.
+    """
+
+    spike_times = list()
+    spike_gids = list()
+    spike_types = list()
+    for file in sorted(glob(str(fname))):
+        spike_trial = np.loadtxt(file, dtype=str)
+        if spike_trial.shape[0] > 0:
+            spike_times += [list(spike_trial[:, 0].astype(float))]
+            spike_gids += [list(spike_trial[:, 1].astype(int))]
+
+            # Note that legacy HNN 'spk.txt' files don't contain a 3rd column
+            # for spike type. If reading a legacy version, validate that a
```

Comment:
```suggestion
            # for spike type. If reading a legacy version, ensure that a
```
Just kind of awkward sounding reading this again.